test/functional/test_framework/wallet.py

```diff
@@ -147,7 +148,7 @@ def get_descriptor(self):
     def get_address(self):
         return self._address
 
-    def get_utxo(self, *, txid: str = '', vout: Optional[int] = None, mark_as_spent=True):
+    def get_utxo(self, *, txid: str = '', vout: Optional[int] = None, mark_as_spent=True) -> dict:
```

Comment:
This PR doesn't seem to require this change, does it? If not, could you mention that in the commit message? That would help reviewers.